src/App.js

```diff
@@ -1,24 +1,24 @@
 import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import List from './components/List';
+import AddItem from './components/AddItem';
+import Navigation from './components/Navigation';
+import './components/main.css';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <BrowserRouter>
+        <div>
+          <Switch>
+            <Route exact path="/" component={List} />
+            <Route exact path="/add" component={AddItem} />
+          </Switch>
+        </div>
+        <div className="navbar">
+          <Navigation />
```

Comment:
I like that you used a Navigation component with NavLinks and everything here. 👍